api/MediaStreamTrack.json

```diff
@@ -29,10 +29,10 @@
             "version_added": "16"
           },
           "safari": {
-            "version_added": "11"
+            "version_added": false
```

Comment:
http://mdn-bcd-collector.appspot.com/tests/api/MediaStreamTrack is similarly complex and I don't think these changes are correct.

Reply:
Looks like there was an issue with the test in regards to importing promises that caused this.  I'm deploying a fix to the collector and have reverted changes.